client/reader/stats.js

```diff
@@ -110,6 +110,15 @@ export function recordTrackForPost( eventName, post = {}, additionalProps = {} )
 	}, additionalProps ) );
 }
 
+export function recordTrainTrackInteract( action, post = {} ) {
+	if ( post.railcar ) {
```

Comment:
I like that idea. Could we have a whitelist of eventnames inside recordTrackForPost?